pom.xml

```diff
@@ -31,6 +33,11 @@
       <artifactId>grpc-stub</artifactId>
       <version>1.21.0</version>
     </dependency>
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
+      <version>1.3.2</version>
+    </dependency>
```

Comment:
Is the javax.annotation-api dependency required for this change? I tried to remove it and rebuild and test with the transfer example worked ok.

Reply:
That's true, I tried it with Java 11 and then you need it for the @Generated annotation, that's a gRPC weirdy: https://github.com/grpc/grpc-java/issues/5343  
But technically, for Java 8 it's not necessary, but also doesn't hurt. As it is now, the Java version can just be switched to 11 in the POM.

Reply:
As I read the issue, it seems to be a problem with Java 9 already, so I would keep it there and document it.

Reply:
Ok, I will go ahead and merge the pr. Thanks for your contribution!